tests/test_cli.py

```diff
@@ -102,6 +104,14 @@
     assert env_name in result.output
     assert env_id in result.output
 
+    filename = ".inmanta"
+    try:
+        result = await cli.run("environment", "save", env_name)
+        assert result.exit_code == 0
```

Comment:
Also add an assertion to verify that the .inmanta file exists and verify the content of the file.